scss/civicrm/contact/pages/_manage-groups.scss

```diff
@@ -104,15 +94,16 @@
       margin-bottom: 20px;
     }
 
-    h3,
-    .accordion,
     .dataTables_wrapper {
+      @include box-shadow($crm-form-layout-shadow-lower);
+      border-radius: 0 0 $border-radius-small $border-radius-small;
```

Comment:
what's this custom border-radius? Shouldn't we have one, global?

Reply:
For both this radius comments, 
No, it depends on where data table sits, if it's looking like panel with some header, we just need border radius on bottom, or we do not need it